.github/workflows/update_release_refs.py

```diff
@@ -0,0 +1,32 @@
+import os
+import sys
+
+import yaml
+
+folder_path = sys.argv[1]
+# Define the folder where YAML files are located
+# folder_path = f"/manifests/{new_version}"  # Replace with the actual folder path
+
+# Get the folder name and use it as the new version
+new_version = os.path.basename(folder_path)
```

Comment:
Instead of this, you should allow users to enter the manifest full path and retrieve the version from the actual yml file like in any other workflow in this repo.

Reply:
This was one of the possible workflow solutions in the issue discussions.
https://github.com/opensearch-project/opensearch-build/issues/2546#issuecomment-1781850069